sdk/digitaltwins/Azure.DigitalTwins.Core/samples/Readme.md

```diff
@@ -1,3 +1,7 @@
+# Introduction
+
+Azure Digital Twins is a developer platform for next-generation IoT solutions that lets you create, run, and manage digital representations of your business environment, securely and efficiently in the cloud. With Azure Digital Twins, creating live operational state representations is quick and cost-effective, and digital representations stay current with real-time data from IoT and other data sources. If you are new to Azure Digital Twins and would like to learn more about the platform, please make sure you check out the Azure Digital Twins [official documentation page](https://docs.microsoft.com/azure/digital-twins/overview).
+
```

Comment:
It is worth mentioning that following that documentation page will help one get started with instance creation, access, etc, and that the guide below will help for various API examples.